datafusion/core/src/execution/context.rs

```diff
@@ -551,12 +551,14 @@ impl SessionContext {
     }
 
     /// Creates a [`DataFrame`] for reading an Avro data source.
-    pub async fn read_avro(
+    pub async fn read_avro_with_multi_paths(
```

Comment:
what do you think about calling this `read_avro_from_paths`? rather than `multi_paths`?